<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmColorPanelHSB</class>
 <widget class="QWidget" name="frmColorPanelHSB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmColorPanelHSB</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="ColorPanelHSB" name="colorPanelHSB"/>
   </item>
   <item>
    <widget class="ColorPanelBar" name="colorPanelBar">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labColor">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorPanelBar</class>
   <extends>QWidget</extends>
   <header>colorpanelbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPanelHSB</class>
   <extends>QWidget</extends>
   <header>colorpanelhsb.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
